src/DssSpell.sol

```diff
@@ -36,62 +41,40 @@ contract DssSpellAction is DssAction {
-    address constant MCD_FLIP_ETH_C = 0x7A67901A68243241EBf66beEB0e7b5395582BF17;
+    // Disable Office Hours
+    function officeHours() public override returns (bool) {
+        return false;
+    }
 
     function actions() public override {
-        // Rates Proposal
-        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", THREE_PCT, true);
-        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", THREE_PT_FIVE_PCT, true);
-        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", FOUR_PT_FIVE_PCT, true);
-        DssExecLib.setIlkStabilityFee("UNIV2ETHUSDT-A", FIVE_PCT, true);
-        DssExecLib.setIlkStabilityFee("UNIV2LINKETH-A", FIVE_PCT, true);
-        DssExecLib.setIlkStabilityFee("UNIV2UNIETH-A", FIVE_PCT, true);
-        DssExecLib.setIlkStabilityFee("UNIV2AAVEETH-A", FIVE_PCT, true);
-        DssExecLib.setIlkStabilityFee("MANA-A", FOUR_PCT, true);
-        DssExecLib.setIlkStabilityFee("UNIV2WBTCDAI-A", TWO_PCT, true);
-        DssExecLib.setIlkStabilityFee("RENBTC-A", FIVE_PCT, true);
-
-        // Onboarding ETH-C
-        CollateralOpts memory ETH_C = CollateralOpts({
-            ilk: "ETH-C",
-            gem: DssExecLib.getChangelogAddress("ETH"),
-            join: MCD_JOIN_ETH_C,
-            flip: MCD_FLIP_ETH_C,
-            pip: DssExecLib.getChangelogAddress("PIP_ETH"),
-            isLiquidatable: true,
-            isOSM: true,
-            whitelistOSM: false,
-            ilkDebtCeiling: 100 * MILLION,
-            minVaultAmount: 2000,
-            maxLiquidationAmount: 50000,
-            liquidationPenalty: 1300,
-            ilkStabilityFee: THREE_PT_FIVE_PCT,
-            bidIncrease: 300,
-            bidDuration: 4 hours,
-            auctionDuration: 4 hours,
-            liquidationRatio: 17500
-        });
-        addNewCollateral(ETH_C);
-        DssExecLib.setIlkAutoLineParameters("ETH-C", 2000 * MILLION, 100 * MILLION, 12 hours);
-
-        DssExecLib.setChangelogAddress("MCD_JOIN_ETH_C", MCD_JOIN_ETH_C);
-        DssExecLib.setChangelogAddress("MCD_FLIP_ETH_C", MCD_FLIP_ETH_C);
-
-        // bump changelog version
-        DssExecLib.setChangelogVersion("1.2.10");
+
+        address ILK_REGISTRY = DssExecLib.getChangelogAddress("ILK_REGISTRY");
+        IlkRegistryLike registry = IlkRegistryLike(ILK_REGISTRY);
+        bytes32[] memory ilks = registry.list();
+
+        // Increase Dust Parameter for most Ilks
+        //
+        // Loop over all ilks
+
+        for (uint256 i = 0; i < ilks.length; i++) {
+            // skip the rest of the loop for the following ilks:
+            //
+            if (ilks[i] == "ETH-B" ||
+                ilks[i] == "PSM-USDC-A"
+            ) { continue; }
+
+            // Increase Dust Paramater
+            //
+            DssExecLib.setIlkMinVaultAmount(ilks[i], 5000);
+        }
+
+
+        // Increase Debt Ceiling
+        DssExecLib.setIlkAutoLineDebtCeiling("ETH-A", 15000 * MILLION);
```

Comment:
Can this be set to `15_000` for readability?

Reply:
Alternatively was thinking you could just use a `BILLION` const.

Reply:
15 * BILLION

Reply:
I'm good with either.